pkg/operator/ceph/pool/status.go

```diff
@@ -81,3 +84,14 @@ func updateStatusInfo(cephBlockPool *cephv1.CephBlockPool) {
 
 	cephBlockPool.Status.Info = m
 }
+
+func (r *ReconcileCephBlockPool) updatePoolID(cephBlockPool *cephv1.CephBlockPool) {
+	poolName := cephBlockPool.ToNamedPoolSpec().Name
+	poolDetails, err := cephclient.GetPoolDetails(r.context, r.clusterInfo, poolName)
```

Comment:
instead of requiring can we get this from,
https://github.com/rook/rook/blob/70c1f27b5e9f215d0010d44808a2fbdc8b5dd887/pkg/operator/ceph/csi/peermap/config.go#L140

it can be returned from https://github.com/rook/rook/blob/70c1f27b5e9f215d0010d44808a2fbdc8b5dd887/pkg/operator/ceph/csi/peermap/config.go#L146

Reply:
we are not returning anything from the `ReconcilePoolIDMap` as of now. And this method only called when mirroring is enabled.  We are updating the pool ID even if mirroring is not enabled.